roles/ceph-osd/tasks/scenarios/lvm.yml

```diff
@@ -1,20 +1,26 @@
 ---
 - name: use ceph-volume to create filestore osds with dedicated journals
   command: "ceph-volume lvm create --filestore --data {{ item.data_vg }}/{{ item.data }} --journal {{ item.journal }}"
+  environment:
+    CEPH_VOLUME_DEBUG: 1
```

Comment:
@andrewschoen do you remember why we ended up setting `CEPH_VOLUME_DEBUG: 1` to all ceph-volume calls?